internal/app/api/v1/server.go

```diff
@@ -46,8 +46,8 @@ func (s kubtestAPI) Init() {
 
 	executions := s.Routes.Group("/executions")
 
-	executions.Get("/executions", s.ListExecutions())
-	executions.Get("/executions/:id", s.GetScriptExecution())
+	executions.Get("/", s.ListExecutions())
+	executions.Get("/:executionID", s.GetScriptExecution())
```

Comment:
It will brake the code in handler 
Additionally keep in mind that both routes (this one and one on top of script resource )  are managed by single handler and Paramus need to be the same

Reply:
It's ok just checked you've fixed this 😁